nucypher/blockchain/eth/interfaces.py

```diff
@@ -584,6 +585,10 @@ def sign_and_broadcast_transaction(self,
         #
 
         try:  # TODO: Handle block confirmation exceptions
+            if confirmations:
+                emitter.message(f'Waiting for {confirmations} confirmations', color='yellow')
+            else:
+                emitter.message(f'Waiting {self.TIMEOUT} seconds for receipt', color='yellow')
```

Comment:
I don't think confirmations and timeout are mutually exclusive. It's a "keep trying until one condition, sufficient confirmations or timeout, happens first", no? If so. we may need to reword.

Perhaps something like:
```suggestion
            waiting_for = 'receipt'
            if confirmations:
                waiting_for = f'{confirmations} confirmations'
            emitter.message(f'Waiting {self.TIMEOUT} seconds for {waiting_for}', color='yellow')
```